bitcoin/chainparams.c

```diff
@@ -0,0 +1,73 @@
+#include "chainparams.h"
+#include <string.h>
+
+#define CHAINPARAMS_NETWORK_COUNT 4
+
+const struct chainparams networks[CHAINPARAMS_NETWORK_COUNT] = {
```

Comment:
You can eliminate CHAINPARAMS_NETWORK_COUNT here, and just use [] to size it by the number of entries.

There's then a helpful ccan/array_size module which gives you ARRAY_SIZE(networks).